### PR TITLE
this is test2

### PR DESCRIPTION
**Pull request by jinchihe**

### CODE REVIEW

**Comment by prow-bot-pli** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/kubernetes-discovery/pipelines-vote-ui/pull/7#" title="Author self-approved">jinchihe</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **vincent-pli** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @vincent-pli` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=kubernetes-discovery%2Fpipelines-vote-ui).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/kubernetes-discovery/pipelines-vote-ui/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by jinchihe** [comment]
cc @xq2005

File: app.py
Comment (vincent-pli) [inline app.py:11]: Should not use a name of human being

**Review by vincent-pli — CHANGES_REQUESTED** [review]

**Comment by prow-bot-pli** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/kubernetes-discovery/pipelines-vote-ui/pull/7#" title="Author self-approved">jinchihe</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **vincent-pli** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @vincent-pli` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=kubernetes-discovery%2Fpipelines-vote-ui).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/kubernetes-discovery/pipelines-vote-ui/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: app.py
Comment (jinchihe) [reply app.py]: Updated, thanks :-)

**Review by xq2005 — COMMENTED** [review]
/lgtm

**Comment by prow-bot-pli** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/kubernetes-discovery/pipelines-vote-ui/pull/7#" title="Author self-approved">jinchihe</a>*, *<a href="https://github.com/kubernetes-discovery/pipelines-vote-ui/pull/7#pullrequestreview-748570084" title="Approved">xq2005</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **vincent-pli** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @vincent-pli` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=kubernetes-discovery%2Fpipelines-vote-ui).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/kubernetes-discovery/pipelines-vote-ui/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by vincent-pli — APPROVED** [review]

**Comment by prow-bot-pli** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/kubernetes-discovery/pipelines-vote-ui/pull/7#" title="Author self-approved">jinchihe</a>*, *<a href="https://github.com/kubernetes-discovery/pipelines-vote-ui/pull/7#issuecomment-914848150" title="Approved">xq2005</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **vincent-pli** after the PR has been reviewed.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=kubernetes-discovery%2Fpipelines-vote-ui).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/kubernetes-discovery/pipelines-vote-ui/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by prow-bot-pli** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by: *<a href="https://github.com/kubernetes-discovery/pipelines-vote-ui/pull/7#" title="Author self-approved">jinchihe</a>*, *<a href="https://github.com/kubernetes-discovery/pipelines-vote-ui/pull/7#issuecomment-914848150" title="Approved">xq2005</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=kubernetes-discovery%2Fpipelines-vote-ui).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/kubernetes-discovery/pipelines-vote-ui/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
